units/Legion/Vehicles/T2 Vehicles/legamcluster.lua

```diff
@@ -158,13 +159,12 @@ return {
 				impulsefactor = 0.5,
 				name = "Cluster Munitions",
 				noselfdamage = true,
-				range = 180,
+				range = 100,
 				rgbcolor = "0.7 0.7 1.0 1.0 1.0 1.0 1.0 1.0",
 				soundhit = "xplomed4",
 				soundhitwet = "splsmed",
 				soundstart = "cannhvy2",
 				weapontype = "Cannon",
-				weaponvelocity = 450,
```

Comment:
Weapondef cleanup could be separate.

Reply:
This is the cluster weapon def, `weaponvelocity` is no-op because `range` is specified.

Reply:
Hasn't it been no-op before already?

Reply:
The cluster weapon def is special, since it's spawned by gadget rather than the unit firing it. `weaponvelocity` only has an effect if `range` is not present, though I don't know why you would ever specify `weaponvelocity` instead of `range`.